qrenderdoc/Code/Interface/QRDInterface.h

```diff
@@ -603,7 +603,8 @@ enum class FollowType : int
   OutputColor,
   OutputDepth,
   ReadWrite,
-  ReadOnly
+  ReadOnly,
+  OutputDepthResolve
```

Comment:
@baldurk Let me know what you think about creating a new enum here for the depth resolve output. To me it made more sense then reusing OutputDepth and forcing resolve to be index 1 when creating the Following object for it.

Reply:
Yeh I think that's fine.